src/routes/local.ts

```diff
@@ -26,7 +27,9 @@ router.get('/satellite', (req, res) => {
         return SatelliteUtilities.getSatelliteByName(req.query.name)
             .then((satelliteDoc) => {
                 if (!satelliteDoc) return res.sendStatus(400)
-                return res.json(_.pick(satelliteDoc, PROPS_TO_SEND_IN_RESPOSNE))
+                const satellitesArray = <Partial<SatelliteInterface>[]> []
+                satelliteDoc.map((satellite) => satellitesArray.push(_.pick(satellite, PROPS_TO_SEND_IN_RESPOSNE)))
```

Comment:
Either change .map to .forEach, or remove .push from inside .map and assign .map output to satellitesArray.